app/kumactl/cmd/generate/generate_certificate.go

```diff
@@ -19,42 +19,54 @@ type generateCertificateContext struct {
 	*kumactl_cmd.RootContext
 
 	args struct {
-		key      string
-		cert     string
-		certType string
+		key                  string
+		cert                 string
+		certType             string
+		controlPlaneHostname []string
 	}
 }
 
 func NewGenerateCertificateCmd(pctx *kumactl_cmd.RootContext) *cobra.Command {
 	ctx := &generateCertificateContext{RootContext: pctx}
 	cmd := &cobra.Command{
-		Use:   "certificate",
-		Short: "Generate certificate",
+		Use:   "tls-certificate",
+		Short: "Generate a TLS certificate",
 		Long:  `Generate self signed key and certificate pair that can be used for example in Dataplane Token Server setup.`,
+		Example: `
+# Generate a TLS certificate for use by an HTTPS server, i.e. by the Dataplane Token server
```

Comment:
Let's fix formatting - there should be 2 paces in the beginning of every line of `Examples`